backpack-lint/src/main/java/net/skyscanner/backpack/lint/check/BpkComposeComponentUsageDetector.kt

```diff
@@ -71,6 +71,14 @@ class BpkComposeComponentUsageDetector : Detector(), SourceCodeScanner {
     val webName: String,
     val componentsToReplace: Set<String>,
   ) {
+    BADGE(
+      fullName = "net.skyscanner.backpack.compose.badge.BpkBadge",
+      webName = "badge",
+      componentsToReplace = setOf(
+        "androidx.compose.material.Badge",
```

Comment:
badge (& badged box) in material are actually something different (it's the badge to use for unread messages or similar) - other than the name there's no similarity